README.rst

```diff
@@ -1643,25 +1643,25 @@ Choose from the list of available rules:
 
   Lambdas not (indirect) referencing ``$this`` must be declared ``static``.
 
-  *Risky rule: risky when using "->bindTo" on lambdas without referencing to ``$this``.*
+  *Risky rule:* risky when using "->bindTo" on lambdas without referencing to ``$this``.
```

Comment:
Unrelated, but shouldn't this be ` ``->bindTo`` `?

Reply:
Yes it should 😁